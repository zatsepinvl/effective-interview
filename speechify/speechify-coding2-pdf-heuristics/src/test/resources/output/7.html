            <html><body>
            <div style="display: flex; flex-direction: row">
            <div style="width: 50%">
            <h1>Expected</h1>
            <hr />
            <h2>Language and Metaphor</h2>
<p>This system of signs may be understood as language-like because it has a consistent structure that requires signs to be sequentially ordered according to who, what, how, and when. “Who” must be signed first to distinguish between performers in the group; “what” must be signed second to determine what content is to be performed; and “when” should always be signed last as this instigates performer contribution. Signs are issued with the Soundpainter standing up with bent knees—the bent knees symbolize preparation and inform the group to “get ready” (Ibid.). “When” gestures are signed in an imaginary “box” in front of the Soundpainter: when the Soundpainter leans into the “box” the performers react to the signs. “How” the material is to be performed is an optional part of the language: the Soundpainter can choose whether to indicate volume and tempo. How content should be played can be imposed, but there is the opportunity to leave some aspects entirely up to the performer. When “how” is determined, it must be signed before “when,” to ensure that it is applied to content.</p>
<p>Although signs have a defined meaning, they are not entirely prescriptive. Like verbal language, which can be tailored to generate a personal style and tone, Soundpainting encourages individual contributions from the performers. Most notably, individual contributions are fostered by using “sculpting” gestures: these are gestures which search for material without determining exactly what is to be performed. For example, the Soundpainter might start a performance by searching for a sound which is not of the painter’s choosing by signing “point to point”: the Soundpainter points at individual performers requesting them to play anything of their choosing as long as they are being pointed at. Each time the performer is pointed at he or she must offer new material. Once the material is offered the Soundpainter can either stop it, or negotiate with the performer to alter and develop the content.</p>
<p>Thompson has acknowledged that at first he had not realized the signing structure had become so refined. During 1974–1997 the structure was used but without name. It was at 1997 residency at Woodstock that Thompson and Sarah Weaver classified this syntax more precisely as:</p>
<p>Who: “whole group” (appendix), “brass” or “dancer” – these “function” gestures can be very specific and relate to individuals.</p>
<p>What: “long tone” (appendix) or “point to point” (appendix) – these “sculpting” gestures specify content, and although it is possible to request something specific (e.g. middle C), if “long tone” is signed without a note name, there is a choice left for the performer, as to which tone they play. What also uses “function” gestures to ask performers to “continue”</p>
<p>or for the Soundpainter to denote “this is,” in order to identify the performed material, for example, as a “memory.”</p>
<p>How: (optional) “volume fader” or “tempo fader” – these “sculpting” gestures impose and change how loud or fast the performers should play.</p>
<p>When: “play” [appendix], “stop”, or “fade in/out” – these “functional” gestures control when performers contribute.</p>
<p>The labeling of gestures seems all-important in establishing their communicative process. The “iconic” status afforded these gestures by Thompson establishes them as signs with direct reference to artistic elements. They are gestures that represent the sound produced through our metaphorical understanding of music as a spatial and temporal art form. By classifying them as “iconic” (Thompson 2010), Thompson recognizes an inherent reference to everyday language accompanying gestures. Examples of signs created to relate to everyday activities include the following: “volume fader,” which is based on a studio reference to amplifier faders in that they can be pushed up and down in a straight line by the fingers; and “play,” which is taken from bowling. The release of the ball is equated to the start of the gesture response, the point at which the performer must react to the Soundpainter’s request. Other, more advanced, gestures correlate to everyday functions: the turning of a key in a lock by a single hand equates to the musical “key” and so allows the Soundpainter to request modulations and specific tonal regions.</p>
<p>Soundpainting emphasizes the problematic lexicon musicians and dancers share in that its language is often insufficient, contradictory or metaphorical. The term Soundpainting itself, for example, is perplexing and demonstrates the metaphorical basis of the gestures as well as the mapping between the audio and visual elements which is integral to this creative process: a gesture is a silent movement, which bears meaning, but is only realized when someone responds by interpreting it in sound and/or movement. Figure 1 illustrates how a dialogue is formed between the Soundpainter and performer: it represents the reciprocal nature of Soundpainting in the reliance of Soundpainter on the ensemble and vice versa.</p>
<p>This model combines the many aspects of communication detailed by Jensenius et al, after Zhao and McNeil, in a collaborative study of interpreting music through bodily engagement (Jensenius et al 2010, 14). They identify three dimensions that offer a useful frame to reassess the gesture types used in Soundpainting. First, gesture is a type of non-verbal communication, as also exposed by Lidov: “[a] true gesture is a precise non-verbal articulation” (Lidov 2006, 30). Second, as Soundpainting sets up a system of gesture</p>
<p><i>heLen JuLia minors</i></p>
<p><i>89</i></p>
            </div>
            <div style="width: 50%">
            <h1>Actual</h1>
            <hr />
            
            </div>
            </div>
            </body></html>