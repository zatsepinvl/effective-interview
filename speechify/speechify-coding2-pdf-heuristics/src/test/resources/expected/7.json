[
  { "type": "Heading", "text": "Language and Metaphor" },
  {
    "type": "Paragraph",
    "text": "This system of signs may be understood as language-like because it has a consistent structure that requires signs to be sequentially ordered according to who, what, how, and when. \u201cWho\u201d must be signed first to distinguish between performers in the group; \u201cwhat\u201d must be signed second to determine what content is to be performed; and \u201cwhen\u201d should always be signed last as this instigates performer contribution. Signs are issued with the Soundpainter standing up with bent knees\u2014the bent knees symbolize preparation and inform the group to \u201cget ready\u201d (Ibid.). \u201cWhen\u201d gestures are signed in an imaginary \u201cbox\u201d in front of the Soundpainter: when the Soundpainter leans into the \u201cbox\u201d the performers react to the signs. \u201cHow\u201d the material is to be performed is an optional part of the language: the Soundpainter can choose whether to indicate volume and tempo. How content should be played can be imposed, but there is the opportunity to leave some aspects entirely up to the performer. When \u201chow\u201d is determined, it must be signed before \u201cwhen,\u201d to ensure that it is applied to content."
  },
  {
    "type": "Paragraph",
    "text": "Although signs have a defined meaning, they are not entirely prescriptive. Like verbal language, which can be tailored to generate a personal style and tone, Soundpainting encourages individual contributions from the performers. Most notably, individual contributions are fostered by using \u201csculpting\u201d gestures: these are gestures which search for material without determining exactly what is to be performed. For example, the Soundpainter might start a performance by searching for a sound which is not of the painter\u2019s choosing by signing \u201cpoint to point\u201d: the Soundpainter points at individual performers requesting them to play anything of their choosing as long as they are being pointed at. Each time the performer is pointed at he or she must offer new material. Once the material is offered the Soundpainter can either stop it, or negotiate with the performer to alter and develop the content."
  },
  {
    "type": "Paragraph",
    "text": "Thompson has acknowledged that at first he had not realized the signing structure had become so refined. During 1974\u20131997 the structure was used but without name. It was at 1997 residency at Woodstock that Thompson and Sarah Weaver classified this syntax more precisely as:"
  },
  {
    "type": "Paragraph",
    "text": "Who: \u201cwhole group\u201d (appendix), \u201cbrass\u201d or \u201cdancer\u201d \u2013 these \u201cfunction\u201d gestures can be very specific and relate to individuals."
  },
  {
    "type": "Paragraph",
    "text": "What: \u201clong tone\u201d (appendix) or \u201cpoint to point\u201d (appendix) \u2013 these \u201csculpting\u201d gestures specify content, and although it is possible to request something specific (e.g. middle C), if \u201clong tone\u201d is signed without a note name, there is a choice left for the performer, as to which tone they play. What also uses \u201cfunction\u201d gestures to ask performers to \u201ccontinue\u201d"
  },
  {
    "type": "Paragraph",
    "text": "or for the Soundpainter to denote \u201cthis is,\u201d in order to identify the performed material, for example, as a \u201cmemory.\u201d"
  },
  {
    "type": "Paragraph",
    "text": "How: (optional) \u201cvolume fader\u201d or \u201ctempo fader\u201d \u2013 these \u201csculpting\u201d gestures impose and change how loud or fast the performers should play."
  },
  {
    "type": "Paragraph",
    "text": "When: \u201cplay\u201d [appendix], \u201cstop\u201d, or \u201cfade in/out\u201d \u2013 these \u201cfunctional\u201d gestures control when performers contribute."
  },
  {
    "type": "Paragraph",
    "text": "The labeling of gestures seems all-important in establishing their communicative process. The \u201ciconic\u201d status afforded these gestures by Thompson establishes them as signs with direct reference to artistic elements. They are gestures that represent the sound produced through our metaphorical understanding of music as a spatial and temporal art form. By classifying them as \u201ciconic\u201d (Thompson 2010), Thompson recognizes an inherent reference to everyday language accompanying gestures. Examples of signs created to relate to everyday activities include the following: \u201cvolume fader,\u201d which is based on a studio reference to amplifier faders in that they can be pushed up and down in a straight line by the fingers; and \u201cplay,\u201d which is taken from bowling. The release of the ball is equated to the start of the gesture response, the point at which the performer must react to the Soundpainter\u2019s request. Other, more advanced, gestures correlate to everyday functions: the turning of a key in a lock by a single hand equates to the musical \u201ckey\u201d and so allows the Soundpainter to request modulations and specific tonal regions."
  },
  {
    "type": "Paragraph",
    "text": "Soundpainting emphasizes the problematic lexicon musicians and dancers share in that its language is often insufficient, contradictory or metaphorical. The term Soundpainting itself, for example, is perplexing and demonstrates the metaphorical basis of the gestures as well as the mapping between the audio and visual elements which is integral to this creative process: a gesture is a silent movement, which bears meaning, but is only realized when someone responds by interpreting it in sound and/or movement. Figure 1 illustrates how a dialogue is formed between the Soundpainter and performer: it represents the reciprocal nature of Soundpainting in the reliance of Soundpainter on the ensemble and vice versa."
  },
  {
    "type": "Paragraph",
    "text": "This model combines the many aspects of communication detailed by Jensenius et al, after Zhao and McNeil, in a collaborative study of interpreting music through bodily engagement (Jensenius et al 2010, 14). They identify three dimensions that offer a useful frame to reassess the gesture types used in Soundpainting. First, gesture is a type of non-verbal communication, as also exposed by Lidov: \u201c[a] true gesture is a precise non-verbal articulation\u201d (Lidov 2006, 30). Second, as Soundpainting sets up a system of gesture"
  },
  { "type": "Marginalia", "text": "heLen JuLia minors" },
  { "type": "Marginalia", "text": "89" }
]
